client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx

```diff
@@ -224,6 +225,8 @@ export default function CampaignItemDetails( props: Props ) {
 		  )
 		: '';
 
+	const isLessThanOneWeek = false; //! is_evergreen && activeDays < 7;
```

Comment:
Don't forget to revert this hardcoded value back to `! is_evergreen && activeDays < 7`

Reply:
holy.. thanks for noticing this.. yes i hardcoded this to test it out.. nice catch. fixed it @sbarbosa